### ISSUE TITLE
How to merge code coverage after parallel runs?

### ISSUE THREAD

**Issue by yann-combarnous** [cypress-io/github-action#326]
So I have looked at this actions' doc, and looked into the cypress/real-world app, but I am not clear on how to address the following scenario:
- I am running Cypress against local CRA dev server, instrumented
- if I turn parallel on, how do I get a summary code coverage file for all parallel runs?

The idea is to be able to publish Lcov report as comment or have a follow-up action on cov summary reports.

**Comment by bahmutov** [comment]
If you are running specs in parallel, then each machine will get a partial coverage summary, and you would need to merge it. Thus every job would probably need to upload its code coverage results, then you would need a job to run after all test jobs. That final job would download all code coverage reports and merge into one report.

You could also use 3rd party service, they usually merge partial code coverage reports for you.
